### PR TITLE
<DEVOPS-227>Scheduler creation

### PR DESCRIPTION
**Pull request by pankajholariyacb**

### CODE REVIEW

File: environment/dev/variables.tfvars [hunk 1]
```diff
@@ -203,3 +203,23 @@ odoo_pubsub_domain    = "https://asia-south1-ebo-dev-321910.cloudfunctions.net/c
...
+   zone     = "Asia/Calcutta"
+   method   = "POST"
+   url      = "https://api.ibo.com/fos-engine/api/v1/user-creation/trigger-fallback"
+  },
+  {
+   sname    = "trigger-online-order-notification-v1"
+   cron     = "	0 */2 * * *"
+   zone     = "Asia/Calcutta"
+   method   = "POST"
+   url      = "https://asia-south1-ibo-prod.cloudfunctions.net/send-orders-notifications?reportType=ONLINE"
```
Comment (Sunil-Jacob) [inline environment/dev/variables.tfvars:223]: You are using the url for `ibo-prod` here , Should not it be DEV? @PankaHolariya

File: environment/dev/variables.tfvars [hunk 2]
```diff
@@ -203,3 +203,23 @@ odoo_pubsub_domain    = "https://asia-south1-ebo-dev-321910.cloudfunctions.net/c
...
+
+additional_cloud_scheduler = [
+  {
+   sname    = "fos-engine-cron-fallback-v1"
+   cron     = "*/2 * * * *"
+   zone     = "Asia/Calcutta"
+   method   = "POST"
+   url      = "https://api.ibo.com/fos-engine/api/v1/user-creation/trigger-fallback"
+  },
+  {
```
Comment (Sunil-Jacob) [inline environment/dev/variables.tfvars:218]: The cron for FOS says
```
   sname    = "fos-engine-cron-fallback-v1"
   cron     = "*/2 * * * *"
```
the cron for trigger says

```
 sname    = "trigger-online-order-notification-v1"
   `cron     = "	0 */2 * * *"`
```

Are we missing a `0` for FOS in the `cron` parameter

File: environment/dev/variables.tfvars [hunk 3]
```diff
@@ -203,3 +203,23 @@ odoo_pubsub_domain    = "https://asia-south1-ebo-dev-321910.cloudfunctions.net/c
...
+
+#### cloud scheduler variable ####
+
+additional_cloud_scheduler = [
+  {
+   sname    = "fos-engine-cron-fallback-v1"
+   cron     = "*/2 * * * *"
+   zone     = "Asia/Calcutta"
+   method   = "POST"
+   url      = "https://api.ibo.com/fos-engine/api/v1/user-creation/trigger-fallback"
```
Comment (Sunil-Jacob) [inline environment/dev/variables.tfvars:216]: `"https://api.ibo.com/fos-engine/api/v1/user-creation/trigger-fallback"` --> Are we using production url for DEV?

**Review by Sunil-Jacob — CHANGES_REQUESTED** [review]
Added comments

File: environment/dev/variables.tfvars (hunk 1 above)
Comment (pankajholariyacb) [reply environment/dev/variables.tfvars:223]: okay sir changing

**Pull request closed without merging**